azkaban-webserver/src/main/java/azkaban/webapp/servlet/AbstractAzkabanServlet.java

```diff
@@ -72,6 +72,12 @@
   private String label;
   private String color;
 
+  /*
+   * The variable switchToPanelPage is in charge of switching on retired schedulePanelDeprecated.vm (old UI)
+   * or the new schedulePanel.vm (new UI). We can configure it in conf for this binary change.
+   */
+  private String switchToPanelPage;
```

Comment:
Rename to "SchedulePanelPageName" ?

Even better, make it "enableNewSchedulerUi" and make it a boolean type?